types/src/state.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use cid::Cid;
+use encoding::repr::*;
+use encoding::tuple::*;
+use serde::{Deserialize, Serialize};
+
+/// Specifies the version of the state tree
+#[derive(Debug, PartialEq, Clone, Copy, PartialOrd, Serialize_repr, Deserialize_repr)]
+#[repr(u64)]
+pub enum StateTreeVersion {
+    /// Corresponds to actors < v2
```

Comment:
Doesnt this actually correspond to the version of Hamt we use?

Reply:
It does! Because it uses the versioned hamt, and that version is tied to actor versions

Reply:
Makes complete sense